### PR TITLE
Document Enhance

### PR DESCRIPTION
**Pull request by hzy46**
Review:

- [ ] new abstract, network issues @mydmdm 
- [ ] assign storage @ydye 
- [x] read-only storage @abuccts 
- [x] upgrade guide @Binyang2014

### CODE REVIEW

File: docs/manual/cluster-admin/upgrade-guide.md [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
-7. [How to Add and Remove Nodes](./how-to-add-and-remove-nodes.md)
-8. [How to use CPU Nodes](./how-to-use-cpu-nodes.md)
-9. [How to Customize Cluster by Plugins](./how-to-customize-cluster-by-plugins.md)
-10. [Troubleshooting](./troubleshooting.md)
-11. [How to Uninstall OpenPAI](./how-to-uninstall-openpai.md)
-12. [Upgrade Guide](./upgrade-guide.md) (this document)
-
-The `v1.0.0` is the first release based on pure Kubernetes. If you want to upgrade older version to `v1.0.0`, please refer to [Installation Guide](./installation-guide.md). Upgrade guide from `v1.0.0` will be added in the future.
\ No newline at end of file
+This upgrade guide is only for those who have installed OpenPAI >= `v1.0.0`, and want to upgrade current cluster to a newer version. For example, upgrade from `v1.0.0` to `v1.1.0`. If you want to upgrade older version before `v1.0.0` to version >= `v1.0.0`. Please refer to [Installation Guide](./installation-guide.md).
```
Comment (Binyang2014) [inline docs/manual/cluster-admin/upgrade-guide.md:3]: Recommend user to use new version of dev-box? Old dev-box may use old version of deploy script and miss some lib.

Comment (hzy46) [reply docs/manual/cluster-admin/upgrade-guide.md:3]: Actually I didn't mention dev-box in current document, will add it.

Comment (hzy46) [reply docs/manual/cluster-admin/upgrade-guide.md:3]: Done.

Comment (hzy46) [reply docs/manual/cluster-admin/upgrade-guide.md:3]: Done.

**Pull request merged**